### DISCUSSION TITLE
AIR app running on macOS can not load external files?

### DISCUSSION THREAD

**Discussion by tthtth123** [airsdk/Adobe-Runtime-Support#2619] [Q&A]
When publishing as Flash Player, it could load and play/show music and image well.
But when I publish in AIR SDK (50.2.2.4), it can not be loaded and errors appear!
<img width="960" alt="Screenshot 2023-04-25 at 16 25 30" src="https://user-images.githubusercontent.com/35505570/234235762-8f37a0b1-3448-4fd8-bf60-ff70dc85755f.png">
<img width="960" alt="Screenshot 2023-04-25 at 16 26 04" src="https://user-images.githubusercontent.com/35505570/234235802-396831d1-b2c7-4aa0-ba28-235ac6771dfb.png">

I am using Adobe Animate 2023 running on macOS (Ventura 13.3.1)
I already add the apps ("adl" and "Untitled-1 aiR.app")  into Full Disk Access.

<img width="547" alt="Screenshot 2023-04-25 at 16 30 49" src="https://user-images.githubusercontent.com/35505570/234236016-97486088-e0a4-4668-a34c-eee3c57dcc24.png">

Do you have any suggestion to solve this?

Here is my code:

> import flash.events.Event; 
> import flash.media.Sound; 
> import flash.net.URLRequest; 
>  
> var s:Sound = new Sound(); 
> s.addEventListener(Event.COMPLETE, onSoundLoaded); 
> var req:URLRequest = new URLRequest("/Users/dt/Documents/DATA/data/g1B03TA.mp3"); 
> s.load(req); 
>  
> function onSoundLoaded(event:Event):void 
> { 
>     var localSound:Sound = event.target as Sound; 
>     localSound.play(); 
> }
> 
> var a:Loader = new Loader();
> a.load(new URLRequest("/Users/dt/Documents/DATA/data/game2-HinhAnhGoiY.png"));
> a.contentLoaderInfo.addEventListener(Event.COMPLETE, done);
> function done(e:Event){
>     var bit:Bitmap = e.target.content;
>     if(bit != null){
>         bit.smoothing = true;
>     }
>     addChild(bit);
> }

**Comment by ajwfrost** [comment]
Does it work if you run this up from ADL in the command line/terminal app?
I think the permissions you've got there for full disk access would also need to be granted to Animate..
